firmware/src/peripheral/flash.h

```diff
@@ -0,0 +1,21 @@
+/********************************************************************
+ * File:    flash.h
+ * Author:  Bryant Gonzaga
+ *
+ *******************************************************************/
+
+#ifndef FLASH_H_
+#define FLASH_H_
+
+#include "common_header.h"
+
+#define ADDR_FLASH_SECTOR_0     ((uint32_t)0x08000000) /* Base @ of Sector 0, 16 Kbytes */
+
+/**
+ *
+ */
+uint32_t write_flash(uint32_t num_of_bytes, uint8_t* data);
```

Comment:
In these kinds of functions buffers usually come first. [Like this](http://man7.org/linux/man-pages/man3/memcpy.3.html). Can you swap them?

Reply:
lolol. got it